rest/config.go

```diff
@@ -731,6 +766,7 @@ func ParseCommandLine(runMode SyncGatewayRunMode) {
 	skipRunModeValidation := flag.Bool("skipRunModeValidation", false, "Skip config validation for runmode (accel vs normal sg)")
 	certpath := flag.String("certpath", "", "Client certificate path")
 	cacertpath := flag.String("cacertpath", "", "Root CA certificate path")
+	tlsMinimumVersion := flag.String("tlsMinimumVersion", DefaultMinimumTLSVersion, "Minimum TLS Version to enforce")
```

Comment:
I don't think we should define a flag for this - we try to rely on JSON config where possible.